spec/assessments/largestKeywordDistanceAssessmentSpec.js

```diff
@@ -7,23 +7,23 @@ const Mark = require( "../../js/values/Mark.js" );
 let keywordDistanceAssessment = new largestKeyWordDistanceAssessment();
 
 describe( "An assessment to check the largest percentage of text in which no keyword occurs", function() {
-	it( "returns a bad score when the largest keyword distance is between more than 40%", function() {
+	it( "returns a bad score when the largest keyword distance is between more than 50%", function() {
 		let mockPaper = new Paper( "string with the keyword", { keyword: "keyword" } );
 		let assessment = keywordDistanceAssessment.getResult( mockPaper, Factory.buildMockResearcher( 55 ), i18n );
 
 		expect( assessment.getScore() ).toEqual( 1 );
 		expect( assessment.getText() ).toEqual( "Large parts of your text do not contain the keyword. Try to distribute the keyword more evenly." );
 	} );
 
-	it( "returns an okay score when the largest keyword distance is between 30 and 40%", function() {
+	it( "returns an okay score when the largest keyword distance is between 40 and 50%", function() {
 		let mockPaper = new Paper( "string with the keyword", { keyword: "keyword" } );
 		let assessment = keywordDistanceAssessment.getResult( mockPaper, Factory.buildMockResearcher( 45 ), i18n );
 
 		expect( assessment.getScore() ).toEqual( 6 );
 		expect( assessment.getText() ).toEqual( "Some parts of your text do not contain the keyword. Try to distribute the keyword more evenly." );
 	} );
 
-	it( "returns an okay score when the largest keyword distance is less than 30%", function() {
+	it( "returns an okay score when the largest keyword distance is less than 40%", function() {
```

Comment:
Apparently a pre-existing mistake: "returns an okay score" should be "returns a good score".

Reply:
Fixed.